### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by oskarrough**
Implemented the usual performance tricks. Small improvements.

Time to interactive did not improve. I think the only way to do it with our current stack is server-side rendering or a service-worker. Both not worth it atm.

Another idea is to lazy-load the R4 player since it's 20% of our bundle size but didn't find an easy, reliable way.

before

before

after

after

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>ExplorerDiscogs</title>
-    <meta name="description" content="">
+    <meta name="description" content="An music explorer for Discogs">
```
Comment (u8887) [inline app/index.html]: `An explorer for Discogs`

File: app/index.html [hunk 2]
```diff
@@ -1,25 +1,24 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>ExplorerDiscogs</title>
-    <meta name="description" content="">
+    <meta name="description" content="An music explorer for Discogs">
```
Comment (u8887) [inline app/index.html]: `An explorer for discogs` or `A music...`

File: app/styles/app.css [hunk 3]
```diff
@@ -85,3 +90,21 @@ radio4000-player .Header-media img {
   height: 100%;
   object-fit: cover;
 }
+
+/**
+ * Visually hide an element, but leave it available for screen readers
+ * @link https://github.com/h5bp/html5-boilerplate/blob/master/dist/css/main.css
+ * @link http://snook.ca/archives/html_and_css/hiding-content-for-accessibility
+ */
+.ScreenReader {
```
Comment (u8887) [inline app/styles/app.css]: lint spaces?

File: app/styles/app.css [hunk 4]
```diff
@@ -58,6 +58,11 @@ nav > * + * {
...
 }
 
+nav h1 {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
 .MediaPlayer {
   position: fixed;
 	z-index: 2;
```
Comment (u8887) [inline app/styles/app.css]: lint

File: package.json [hunk 5]
```diff
@@ -34,10 +35,11 @@
     "ember-data": "~3.1.0",
     "ember-export-application-global": "^2.0.0",
     "ember-fetch": "^3.4.5",
+    "ember-inline-css": "^0.0.10",
```
Comment (u8887) [inline package.json]: sure we need that?

Comment (oskarrough) [reply package.json]: no, I'll get rid of it

**Pull request merged**